deploy/examples/common.yaml

```diff
@@ -91,6 +97,9 @@ rules:
   - apiGroups: [""]
     resources: ["serviceaccounts/token"]
     verbs: ["create"]
+  - apiGroups: [""]
```

Comment:
do we need this change?

Reply:
yes https://github.com/rook/rook/issues/10935